test/spec/modules/targetVideoBidAdapter_spec.js

```diff
@@ -0,0 +1,31 @@
+import { spec } from '../../../modules/targetVideoBidAdapter.js'
```

Comment:
The code coverage for this adapter is low. Please add tests for `interpretResponse`.

Reply:
Added tests.